interfaces/builtin/serial_port_test.go

```diff
@@ -82,10 +82,10 @@ slots:
         path: path
     bad-path-2:
         interface: serial-port
-        path: /dev/tty0
+        path: /dev/tty
```

Comment:
The /dev/tty0 should be filter out as bad-path if we have regex "^/dev/tty[a-zA-Z0-9]+[0-9]+$".